Mathlib/Data/Finsupp/Defs.lean

```diff
@@ -673,6 +678,23 @@ theorem erase_zero (a : α) : erase a (0 : α →₀ M) = 0 := by
   classical rw [← support_eq_empty, support_erase, support_zero, erase_empty]
 #align finsupp.erase_zero Finsupp.erase_zero
 
+theorem erase_eq_update (f : α →₀ M) (a : α) : f.erase a = update f a 0 :=
```

Comment:
```suggestion
theorem erase_eq_update_zero (f : α →₀ M) (a : α) : f.erase a = update f a 0 :=
```
for discoverability. I would even consider
```suggestion
@[simp]
theorem update_zero_eq_erase (f : α →₀ M) (a : α) : f.erase a = update f a 0 :=
```

Reply:
Did you mean to flip the direction in that last one?

Reply:
Note that there was already discussion about this in https://github.com/leanprover-community/mathlib4/pull/9316#discussion_r1437765537

Reply:
Yep, sorry, I meant it flipped